packages/jest-circus/types.js

```diff
@@ -89,9 +116,15 @@ export type Event =
     |}
   | {|
       name: 'run_finish',
+    |}
+  | {|
+      // Any unhandled error that happened outside of test/hooks (unless it is
+      // an `afterAll` hook)
+      name: 'error',
+      error: Exception,
     |};
 
-export type TestStatus = 'pass' | 'fail' | 'skip';
+export type TestStatus = 'skip' | 'done';
```

Comment:
i removed `fail` status, since it's a derived value that can be calculated from `test.errors.length`.
All we need to know is whether it was skipped or finished running